be/src/controlers/rest.router.ts

```diff
@@ -1,22 +1,53 @@
-// @ts-ignore
 import {Request, Response, Router} from 'express';
+import {vocabularyService} from "../services/vocabulary.service";
+import {Vocabulary} from "../models/dbo.models";
+import {Vocabulary as VocabularyDTO} from "../generated/models/Vocabulary";
 
 const router: Router = Router();
 
+function vocabDto2Dbo(dto: VocabularyDTO): Vocabulary {
+  const vocab: Vocabulary = {
+    _id: undefined,
```

Comment:
Is setting this to undefined ok, @ja-fra ?

Reply:
This is on purpose: `_id` (like `created` and `lastModified`) are defined readonly, so they must not be changed by the user/client. Therefore the values passed in through the AP are ignored.